scrunch/tests/test_datasets.py

```diff
@@ -250,6 +256,17 @@ def test_edit_dataset(self):
     def process_expr_side_effect(self, expr, ds):
         return expr
 
+    def test_replace_from_csv(self):
+        ds_mock = self._dataset_mock()
+        ds = StreamingDataset(ds_mock)
+        assert ds.resource.body.get('streaming') == 'streaming'
```

Comment:
Also test that a dataset that is with `streaming == no` gets back to its original state.